cachify.php

```diff
@@ -91,7 +91,7 @@
  * @param string $class_name the class name.
  */
 function cachify_autoload( $class_name ) {
-	if ( in_array( $class_name, array( 'Cachify', 'Cachify_APC', 'Cachify_DB', 'Cachify_HDD', 'Cachify_MEMCACHED', 'Cachify_REDIS', 'Cachify_CLI' ), true ) ) {
+	if ( in_array( $class_name, array( 'Cachify', 'Cachify_APC', 'Cachify_DB', 'Cachify_HDD', 'Cachify_MEMCACHED', 'Cachify_REDIS', 'Cachify_CLI', 'Cachify_Interface' ), true ) ) {
```

Comment:
I would prefer alphabetical order here, although `CLI` is any outlier already.

Reply:
Missed totally, that this was in alphabetical order. Thanks!